VikingQuiz.Api/Repositories/UserRepo.cs

```diff
@@ -57,6 +57,11 @@ public User GetUserById(int id)
             return ctx.User.FirstOrDefault(x => x.Id == id);
         }
 
+        public bool ExistsById(int id)
```

Comment:
This should be rename to something more meaningful. ExistsById doesn`t suggest what should exist...